dashboard/src/components/Config/AppRepoList/AppRepoForm.tsx

```diff
@@ -267,6 +267,11 @@ export function AppRepoForm(props: IAppRepoFormProps) {
     return message;
   };
 
+  /* Only when using a namespace different than the Kubeapps namespace (Global)
+    the repository can be associated with Docker Registry Credentials since
+    the pull secret won't be available in all namespaces */
```

Comment:
Could we display this information in a tooltip or similar? Perhaps it's just me, but I felt lost when this button was not appearing until I noticed I was in the Kubeapps namespace.

Reply:
we can maybe show and disable it in the `kubeapps` namespace, with a tooltip (but I would do that in a different PR).